recipes/qtxlsxwriter/all/conandata.yml

```
@@ -0,0 +1,10 @@
+sources:
+  "0.3.0":
+    - url: "https://github.com/dbzhang800/QtXlsxWriter/archive/v0.3.0.zip"
+      sha256: "e665317de4f1551936d519781ab8f96b6738926595f3f26ecb6c50db4ff0419e"
+    - url: "https://raw.githubusercontent.com/dbzhang800/QtXlsxWriter/master/LICENSE"
```

Better to use the commit as well, just in case it is moved, renamed, changed,... This is the commit when the file was added, I suppose sha256 will be the same, but I haven't checked.

```suggestion
    - url: "https://raw.githubusercontent.com/dbzhang800/QtXlsxWriter/d013edc9cf39b450f035528114b349a70b1507ba/LICENSE"
```

---

[Done.](https://github.com/conan-io/conan-center-index/pull/4838/commits/3bf683f258b5fd6056c397bf25113256d6392f94)

> It would be nice to fix Qt5... have you tried this library using Qt6?

[The official QtXlsxWriter documentation](http://qtxlsx.debao.me) mentions that only Qt5 is supported, and taking into account that Qt6 requires C++17 I think that it's not a good idea to restrict the supported version of the standard since this library can work with C++11. I can add the supported Qt version as an option but I'm not sure if it is worth the effort.